sdk/kfserving/api/kf_serving_client.py

```diff
@@ -22,11 +22,11 @@
 class KFServingClient(object):
     """KFServing Apis."""
 
-    def __init__(self):
+    def __init__(self, kube_config=None):
         if utils.is_running_in_k8s():
             config.load_incluster_config()
         else:
-            config.load_kube_config()
+            config.load_kube_config(config_file=kube_config)
```

Comment:
maybe not right place for my comment:
KFServingClient can only work for `constants.KFSERVING_VERSION` KFservice now. To avoid refactoring here when new version KFservice comes, I suggest that we should make kfservice version as an argument at least.

Reply:
@hougangliu I think that's two things. and in fact, I think the KFServingClient APIs is not closely related with KFservice Version. Could you please log another ticket to describe your points? I can address that in another ticket. Thanks a lot!